### DISCUSSION TITLE
Welcome to haxo-hw Discussions!

### DISCUSSION THREAD

**Discussion by jcard0na** [cardonabits/haxo-hw#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ddodge40** [comment]
Don Dodge 
W6SNX ham radio 
Electronic projects with Grandkids
Please send where I can download a zipped copy of the gerber files for the xaxaphone

**Reply by jcard0na** [reply]
Hi @ddodge40 

I just committed them.  See #9.
